### PR TITLE
removed dead services

### PR DESCRIPTION
**Pull request by jbpratt78**

### CODE REVIEW

**Comment by KenanY** [comment]
We may want to remove these from `api/src/Channel.h` too while we're at it. cc @slugalisk to confirm that's okay.

**Comment by slugalisk** [comment]
Thanks for updating the backend channel list! It looks like we can get channel status without an oauth token from the mixer API ex. https://mixer.com/api/v1/channels/ForzaMotorsport https://gist.github.com/slugalisk/047e6db341c68762548e01a154d1f3a8

**Comment by xDashh** [comment]
https://github.com/MemeLabs/Rustla2/pull/92/files#diff-28b937b0b2e194ed5b7eb2522ed8ab40R21 removal is missing I think.

File: src/components/Checkbox.jsx [hunk 1]
```diff
@@ -1,26 +1,22 @@
+
```
Comment (KenanY) [inline src/components/Checkbox.jsx]: I think all of the changes in this file should be reverted.

**Comment by KenanY** [comment]
Also just to make sure, with this PR mixer streams will always appear offline? (Fine by me, just making sure that we are aware and okay with this so it doesn't surprise us post-merge.)

**Comment by slugalisk** [comment]
There's a note above with the API endpoints where we can get Mixer stream state. I can write something to check it if no one else wants to.

File: src/components/Checkbox.jsx [hunk 2]
```diff
@@ -4,42 +4,35 @@ import classNames from 'classnames';
 import compose from 'recompose/compose';
 import withHandlers from 'recompose/withHandlers';
 import withState from 'recompose/withState';
-
```
Comment (KenanY) [inline src/components/Checkbox.jsx]: Commenting again on this because GitHub is hiding the previous comment: let's undo _all_ of the changes to this file since they are unrelated to the rest of the PR.

File: src/components/StreamEmbed.jsx [hunk 3]
```diff
@@ -35,21 +33,16 @@ const getSrc = (channel: string, service: string): string | null => {
...
     case 'youtube':
       return `https://www.youtube.com/embed/${channel}?autoplay=1`;
-    case 'afreeca':
-      return `https://play.afreecatv.com/${channel}/embed`;
     default:
       return null;
   }
 };
 
 type Service =
```
Comment (KenanY) [inline src/components/StreamEmbed.jsx]: Should `'mixer'` be added to this list?

File: src/util/supported-chats.js [hunk 4]
```diff
@@ -1,13 +1,14 @@
...
   'twitch-vod': channel => `https://www.twitch.tv/embed/${channel}/chat?darkpopout`,
   'twitch': channel => `https://www.twitch.tv/embed/${channel}/chat?darkpopout`,
-  'ustream': channel => `https://www.ustream.tv/socialstream/${channel}`,
   'vaughn': channel => `https://vaughnlive.tv/popout/chat/${channel}`,
   'youtube': channel => `https://youtube.com/live_chat?v=${channel}&embed_domain=${process.env.NODE_ENV === 'production' ? 'strims.gg' : 'localhost'}`,
 };
 
 export default supportedChats;
-export const supportedChatServices = new Set<string>(Object.keys(supportedChats));
+export const supportedChatServices = new Set < string > (Object.keys(supportedChats));
```
Comment (KenanY) [inline src/util/supported-chats.js]: Undo this change please.

File: src/util/supported-chats.js [hunk 5]
```diff
@@ -1,13 +1,14 @@
...
   'twitch-vod': channel => `https://www.twitch.tv/embed/${channel}/chat?darkpopout`,
   'twitch': channel => `https://www.twitch.tv/embed/${channel}/chat?darkpopout`,
-  'ustream': channel => `https://www.ustream.tv/socialstream/${channel}`,
   'vaughn': channel => `https://vaughnlive.tv/popout/chat/${channel}`,
   'youtube': channel => `https://youtube.com/live_chat?v=${channel}&embed_domain=${process.env.NODE_ENV === 'production' ? 'strims.gg' : 'localhost'}`,
 };
 
 export default supportedChats;
-export const supportedChatServices = new Set<string>(Object.keys(supportedChats));
+export const supportedChatServices = new Set <string> (Object.keys(supportedChats));
```
Comment (KenanY) [inline src/util/supported-chats.js]: This should remain as it was: `new Set<string>(`

**Comment by KenanY** [comment]
Squashed into a single commit, reworded the commit message, made some minor changes, and merged as https://github.com/MemeLabs/Rustla2/commit/0e65bd975cef7f55bf36daa39585fb8d99a897b2. Thanks!
